CHANGELOG.rst

```diff
@@ -13,6 +13,7 @@ Fixed
 ~~~~~
 
 * Fix unreachable join when an inbound task fails as defined in the task transition. (bug fix)
+* Throw exception if YAML of workflow definition contains duplicate keys (i.e. task name). (bug fix)
```

Comment:
```suggestion
* Throw exception if YAML if workflow definition contains duplicate keys (i.e. task name). (bug fix)
```

Reply:
The change to `if` seems awkward. Are you sure?

Reply:
Nix the preposition entirely?
```suggestion
* Throw exception if workflow definition YAML dictionaries contains duplicate keys (i.e. task name). (bug fix)
```

Reply:
Ok revised it. Let's focus on the code here. You guys have too much time?

Reply:
lol. Yeah. I misread that the first time through. Sorry for the bikeshedding...